oscar_python/client.py

```diff
@@ -132,7 +132,9 @@ def _check_fdl_definition(self, fdl_path):
                     except KeyError as err:
                         raise Exception("FDL clusterID does not match current clusterID: {0}".format(err))
                     try:
-                        with open(svc["script"]) as s:
+                        fdl_directory = os.path.dirname(fdl_path)
+                        script_path = fdl_directory + "/" + svc["script"]
```

Comment:
Make sure that if the user specifies an absolute path for the shell script in the FDL this is respected. If it is a relative path, then you join fdl_directory with the relative path.

Reply:
Should we specify in the documentation that the script should be in a relative path from the fdl file?